app/views/shared/_navbar.html.erb

```diff
@@ -1,6 +1,6 @@
 <nav class="navbar navbar-expand-lg bg-transparent">
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
-        <img src="favicon.ico" alt="">
+        <img id="nav-logo" src="favicon.ico" alt="">
```

Comment:
Sorry I havent pushed my code which has this correction, but this needs to be a image tag (ie. <%= image_tag %>

Reply:
I see! I can remove the id I added for now, and after your code is merged I can put it back in with the proper formatting